src/link.js

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 
-const Link = ({title, url, handleClick}) => (
+import { extract } from "./utils";
 
-  <button onClick={() => handleClick(url)}>{title}</button>
+
+class Link {
+
+    constructor({href, meta}, text = '') {
+        this.href = href;
+        this.text = extract(meta, 'linkParams.title', text);
+    }
+
+    static parseLinks(links) {
+        return Object.keys(links).reduce((parsed, key) => Object.assign(parsed, {
+            [key]: new Link(links[key], key),
```

Comment:
This is interesting- instantiating the class in a static class method?

Reply:
Yeah, that seemed like the most relevant place to put it. This is probably one of those things that doesn't "look like JavaScript", but I like this pattern from other OO languages (i.e. the static "create" method vs always using the constructor)